scss/tech.scss

```diff
@@ -1,25 +1,41 @@
-.tech-container {
-  display: flex;
-  flex-wrap: wrap;
-  padding: 10px 70px;
-  width: 100%;
+.tech-grid-container {
+  display: grid;
+  grid-template-areas: 
```

Comment:
define this without grid areas. Just setting the first element the height of two rows

Reply:
I used grid-column-start, grid-column-end, grid-row-start and grid-row-end but I don't know if it's what you mean